lib/index.js

```diff
@@ -30,37 +35,49 @@ const defaults = {
  */
 
 function StickyInlines(opts) {
-  opts = Object.assign({}, defaults, opts)
-  const { allowedTypes, bannedTypes, hasStickyBoundaries, canBeEmpty } = opts
+  opts = Object.assign({}, defaults, opts);
+  const { allowedTypes, bannedTypes, hasStickyBoundaries, canBeEmpty } = opts;
 
-  if (allowedTypes && !Array.isArray(allowedTypes)) { console.warn('slate-sticky-inline: allowedTypes must be an Array of Strings') }
-  if (!Array.isArray(bannedTypes)) { console.warn('slate-sticky-inlines: bannedTypes must be an Array of Strings') }
+  if (allowedTypes && !Array.isArray(allowedTypes)) {
+    console.warn(
+      'slate-sticky-inline: allowedTypes must be an Array of Strings'
+    );
+  }
+  if (!Array.isArray(bannedTypes)) {
+    console.warn(
+      'slate-sticky-inlines: bannedTypes must be an Array of Strings'
+    );
+  }
 
   /**
    * Keydown entry point.
    *
    * @param {Event} event
    * @param {Change} change
-   * @param {Editor} editor
+   * @param {Next} next
    * @return {Change}
    */
 
```

Comment:
For consistency with the argument list passed to onKeyDown handlers, please rename change to editor

Reply:
I think this is resolved now